.github/test.config.ts

```diff
@@ -17,3 +17,4 @@ export const accounts: Account[] = [
 	},
 ];
 export const clientUrl = 'http://localhost:8545';
+export const clientWsUrl = 'ws://localhost:8545';
```

Comment:
since we aren't using the dockerfile anymore it shouldnt be in the .github folder. i think the `test.config.ts` should be moved back to `providers-http/test/integration`

Reply:
but for ganache, we use these accounts to refill any wallets we want.

Reply:
I think we fix it when will merge @nazarhussain PR